src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -113,7 +113,7 @@ public function getProxyCode(Definition $definition)
         }
 
         if (version_compare(self::getProxyManagerVersion(), '2.5', '<')) {
-            $code = str_replace(' \Closure::bind(function ', ' \Closure::bind(static function ', $code);
+            $code = preg_replace('/ \Closure::bind(function ((?:& )?\(\$instance(?:, \$value)?\))/', ' \Closure::bind(static function \1', $code);
```

Comment:
I think there is still something wrong:

```
In ProxyDumper.php line 116:

  [ErrorException]
  Warning: preg_replace(): Compilation failed: missing closing parenthesis at offset 62
```

The parenthesis are not properly escaped.

Reply:
better now?

Reply:
Yes! Cant confirm whether the related bug is fixed as it seems I am hitting a different bug than the reporter.